shared/js/background.js

```diff
@@ -16,12 +16,15 @@
 
 
 var debugRequest = false
-var trackers = require('trackers')
-var utils = require('utils')
-var settings = require('settings')
-var stats = require('stats')
```

Comment:
What's going on with stats here?

Reply:
Oh, good question. I forgot I removed this.

It's not a module that exists anywhere, nor is it used in this file. It wasn't erroring out before, because with `require.scopes`, `stats` just ends up undefined. Browserify throws an error when you try and require a non-existent module.